CONTRIBUTING.md

```diff
@@ -181,13 +70,13 @@ git push
 
 In the following a few recommendations are outlined which should help you to get familiar with development process in the Acts project.
 
-1. **Each development its own branch!**
-Branching in git is simple, it is fun and it helps you keep your working copy clean. Therefore, you should start a new branch for every development. All work which is logically/conceptually linked should happen in one branch. Keep your branches short. This helps immensly to understand the git history if you need to look at it in the future.
+1. **Each development in its own branch of your private fork!**
+Write access for developers has been disabled for developers on acts-project. Therefore, always start by creating your own fork and creating branches there. You should start a new branch for every development and all work which is logically/conceptually linked should happen in one branch. Try to keep your branches short. This helps immensly to understand the git history if you need to look at it in the future and helps reviewers of your code.
 If projects are complex (e.g. large code refactoring or complex new features), you may want to use _sub_-branches from the main development branch as illustrated in the picture below.
 
 <img src="doc/figures/sub_dev.png" alt="workflow for large feature">
```

Comment:
Honestly I think these images are more confusing than they are helpful, but I'll leave that up to you.

Reply:
removed